cli/src/commands/mod.rs

```diff
@@ -3004,6 +3004,7 @@ fn description_template_for_commit(
 
 fn description_template_for_cmd_split(
     ui: &Ui,
+    command: &CommandHelper,
```

Comment:
We generally pass in `&UserSettings` instead, if that's all that's needed. I don't know if there's much reason to do that, but let's do it for consistency for now.

Reply:
Refactored a few other instances as well.